lib/index.js

```diff
@@ -228,6 +228,52 @@ class Linter {
 
     return message;
   }
+
+  reportGitHubActionAnnotations(errors) {
+    if (!process.env.GITHUB_ACTIONS || process.env.DISABLE_GITHUB_ACTIONS_ANNOTATIONS) {
```

Comment:
@runspired I included `DISABLE_GITHUB_ACTIONS_ANNOTATIONS` just for you 😉

Reply:
Thanks!